recipes/DABEST/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "dabest" %}
+{% set version = "0.1.0" %}
+{% set sha256 = "8147e304ebc587923f0e2692ff68eb33e5abba81d0b93994da1d14cc4479dec4" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
```

Comment:
Also, this package might be a candidate for `noarch: python`

http://conda-forge.readthedocs.io/en/latest/meta.html#building-noarch-packages

Reply:
Ah, thanks for this!